oak_functions/examples/metrics/config.toml

```diff
@@ -0,0 +1,12 @@
+lookup_data_url = "https://storage.googleapis.com/oak_lookup_data/lookup_data_weather"
+lookup_data_download_period = "10m"
+worker_threads = 4
+
+[metrics]
+epsilon = 1.0
+batch_size = 20
+allowed_labels = ["a"]
```

Comment:
I kind of feel that this should be under policy, since it affects the privacy guarantees of the server, and it would be nice if the client only needs to look at the policy section in order to figure out what the guarantees are, rather than looking at fields spread out in the file. Anyways maybe we can refactor things once something like https://github.com/project-oak/oak/pull/2247#issuecomment-891227572 is implemented.